### DISCUSSION TITLE
How to select species - structure clustering - for deep comparative questions

### DISCUSSION THREAD

**Discussion by 000generic** [hahnlab/CAFE5#202] [Q&A] [answered]
Hi!

I'm new to Cafe5 and excited to use it. I'm working on the genome of a gastropod - and interested in tracking origins of its gene families shallow and deep across gastropods, molluscs, animals and eukaryotes.  To do begin this analysis, I used OrthoFinder to cluster the genomes of 67 species including around 20 different animal phyla, 15 different unicellular eukaryotes, and a number of species per class within molluscs. 

Having done that - as I now understand it, Cafe is able to assess gene family birth and death - given a clustering/gene families and a tree for a given set of species, it then evaluates gene family gains/losses statistically - but can only work with gene families that were present in the last common ancestor of all of the species in the tree. 

If I am interested in say 5 different major time points of evolution/clades leading up to my species of interest - gastropods, molluscs, bilaterians, metazoans, holozoans

Can I 

1) use a clustering of all 67 species, which includes holozoans and two mycozoan outgroup species - and submit this to Cafe5 and evaluate gains and losses on the nodes for the major clades of interest - not sure but I have the sense after reading through github that this will only track gene family gain and loss from the very deep last common ancestor of holozoans-mycozoans - meaning most gene families in animals (including all gene families unique to animals or gastropods, for example) will not/should not be evaluated in this analysis.

OR

2) I should I do a series of orthofinder clusterings - one composed only of the gastropod species (no outgroup), one composed only of the mollusc species (no outgroup), one composed only of the bilaterian species (no outgroup) etc - and then submit each of these to Cafe5 for separate analyses?

OR

3) Can I use just the initial clustering of all 67 species - and then for a given subset of species - lets say all the mollusc species - identify orthogroups/gene families in the OrthoFinder clusters that include either 1) at least one mollusc and one non-mollusc species or 2) presence on either side of the deepest node in the mollusc tree - both of which would imply the gene family was present in the last common ancestor of molluscs - and then supply this set of gene families to Cafe5.

If I am interested in gene gain / gain loss throughout the tree - and number 2 or 3 is the correct approach - should I then do this operation for each internal node of the species tree - and then add changes per node per analysis - and then make things non-redunant somehow (?) - to get a global number per node of the species tree?  

Or what would be advised for this type of framing? Any guidance on this would be greatly appreciated!  

Thank you very much, Eric

**Comment by hahnlab-user** [comment, marked as answer]
Hello Eric,

Approaches 2 or 3 are probably the best to use. CAFE can filter out families that it evaluates as "not present" in the root, but of course it would be better if you did this yourself. As for the last point: that seems like a lot of work, and probably won't add too many additional families when carrying out additional clustering on nearby nodes.

One last thing: some of these timeframes are almost certainly too deep for CAFE to analyze. The root of metazoa, for instance, has got to be beyond the lambda*t <1 limit. So I wouldn't even start with holozoans and mycozoans.


cheers,
Matt
